lms/djangoapps/verify_student/tests/test_signals.py

```diff
@@ -0,0 +1,49 @@
+"""
+Unit tests for the course_mode signals
+"""
+
+from datetime import datetime, timedelta
+
+from pytz import UTC
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+from lms.djangoapps.verify_student.models import VerificationDeadline
+from lms.djangoapps.verify_student.signals import _listen_for_course_publish
+
+
+class CourseModeSignalTest(ModuleStoreTestCase):
+    """
+    Tests for the CourseMode model
```

Comment:
Similarly, docstring should refer to signals testing, not the model. Also, presumably this should be named `VerificationDeadlineSignalsTest`.